master/buildbot/test/unit/test_www_oauth.py

```diff
@@ -210,6 +216,144 @@ def test_GithubAcceptToken(self):
+            {
+                'data': {
+                    'viewer': {
+                        'organizations': {
+                            'edges': [
+                                {
+                                    'node': {
+                                        'login': 'hello'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'login': 'grp'
+                                    }
+                                }
+                            ]
+                        },
+                        'login': 'bar',
+                        'email': 'bar@foo',
+                        'name': 'foo bar'
+                    }
+                }
+            },
+            {
+                'data': {
+                    'hello': {
+                        'teams': {
+                            'edges': [
+                                {
+                                    'node': {
+                                        'name': 'developers',
+                                        'slug': 'develpers'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'name': 'contributors',
+                                        'slug': 'contributors'
+                                    }
+                                }
+                            ]
+                        }
+                    },
+                    'grp': {
+                        'teams': {
+                            'edges': [
+                                {
+                                    'node': {
+                                        'name': 'developers',
+                                        'slug': 'develpers'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'name': 'contributors',
+                                        'slug': 'contributors'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'name': 'committers',
+                                        'slug': 'committers'
+                                    }
+                                },
+                            ]
+                        }
+                    },
+                }
+            }
+        ])
+        res = yield self.githubAuth_v4_teams.acceptToken("TOK3N")
+        self.assertEqual({'email': 'bar@foo',
+                          'username': 'bar',
+                          'groups': ["hello", "grp"],
+                          'teams': [
```

Comment:
The thing is the notion of Team is very bound to Github, and the only auth plugin which would implement this. This is why I would prefer not to introduce a new concept unless it is necessary.

I think '/' is already a clean separation. You then can use  regular expression to give rights to grp/* or just grp or grp/foo/*